lib/tasks/world_taxon_update.rake

```diff
@@ -0,0 +1,156 @@
+# This task is to update the titles of each worldwide taxon to include the country name they relate to.
+# This work is necessary to meet the following WCAG guidelines:
+# https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html
+# https://www.w3.org/WAI/WCAG22/Techniques/failures/F25
+# For accessibility reasons, taxon titles need to be more unique. Having multiple versions of pages for different
+# countries all titled e.g. 'Trade and invest' goes against WCAG guidelines.
+
+ROOT_CONTENT_ID = "91b8ef20-74e7-4552-880c-50e6d73c2ff9".freeze
+
+namespace :worldwide do
+  desc "Update worldwide taxon titles (external name) to include the name of the country they relate to"
+
+  task :add_country_name_to_title, %i[log_file_path] => :environment do |_, args|
+    log_file = nil
+    if args[:log_file_path]
+      log_file = File.open(args[:log_file_path], "w")
+      log_file.puts("Updating each worldwide taxon to include country name in their title")
+    end
+    puts("Updating each worldwide taxon to include country name in their title")
+
+    total_taxon_updates = 0
+
+    # Build a taxonomy tree with the grandparent
+    # (common ancestor e.g. /world/all - Help and services around the world) as the root
+    taxonomy = Taxonomy::ExpandedTaxonomy.new(ROOT_CONTENT_ID).build.child_expansion
+    log_file&.puts("Taxonomy has size #{taxonomy.tree.size}")
+    puts("Taxonomy has size #{taxonomy.tree.size}")
```

Comment:
Consider refactoring this bit of a code (and similar occurrences below)
```
    log_file&.puts(message)
    puts(message)
```
into method to avoid duplications